weaviate/batch/crud_batch.py

```diff
@@ -672,41 +750,164 @@ def create_references(self) -> list:
         """
 
         if len(self._reference_batch) != 0:
+            message = (
+                "You are manually batching this means you are NOT using the client's "
+                "built-in multi-threading. Setting `batch_size` in `client.batch.configure()` "
+                "to an int value will enabled this. Also see: https://weaviate.io/developers/"
+                "weaviate/current/restful-api-references/batch.html#example-request-1"
+            )
+            warnings.warn(message, RuntimeWarning, stacklevel=1)
```

Comment:
have this as an extra method? We could have an "errors.py" file that contains all these messages

Reply:
Sounds good